### DISCUSSION TITLE
有提供api来获取zoomlevel吗

### DISCUSSION THREAD

**Discussion by kankantian** [zouyaoji/vue-cesium#481] [Q&A]
1.正常情况下如何获取层级 2.使用viewer.scene.camera.flyTo时又如何获取层级

**Comment by zouyaoji** [comment]
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/utils/cesium-helpers.ts#L1085

试试

**Reply by kankantian** [reply]
已解决，谢谢！
